spec/features/owner_replies_to_reviews_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature 'owner replies to reviews', %{
+  As a vendor ownder
+  I want to reply to reviews on my business
+  So that users can see my comments
+} do
+  # Acceptance Criteria
+  # * If I'm signed in as a member that owns a business
+  # * I should be able to respond to reviews on my vendor profile page
+
+  # * If I'm signed in but do not own the business
+  # * I should not be able to respond to reviews on the vendor profile page
+
+  scenario 'vendor owner can reply to reviews' do
+    member = FactoryGirl.create(:user)
```

Comment:
maybe rename this variable to `owner` to differentiate it from the normal `member` you use in your last test